yt/yt/core/concurrency/throughput_throttler.cpp

```diff
@@ -400,6 +402,8 @@ class TReconfigurableThroughputThrottler
         if (deltaAvailable == 0) {
             return;
         }
+        // The delta computed above is zero if the limit is zero.
+        YT_VERIFY(limit > 0);
```

Comment:
Did you mean "`deltaAvailable` to be positive even when `limit == 0`" in the first statement? Float-shenanigans? 
As for the second point, I agree, it would be much clearer. Maybe @coteeq wants to do it along with the other fixes?

Reply:
> Did you mean "deltaAvailable to be positive even when limit == 0" in the first statement? Float-shenanigans?

It seems impossible, since we literally multiply by `limit` [here](https://github.com/ytsaurus/ytsaurus/blob/b3092d5238a9a1e8d71411f0cfd4dcb7120ca28e/yt/yt/core/concurrency/throughput_throttler.cpp#L350) and I don't know any floating-point issues, which can break this.